wallettool/src/main/graal/reflect-config.json

```diff
@@ -0,0 +1,365 @@
+[
```

Comment:
Are these files generated? If yes, from what source?

Are they even needed? Afair the previous version of your PR didn't have them and judging by the GitHub Actions reports it seemed to work.

Reply:
These files were generated by using `gradle -Pagent run` where the `run` task was configured in such a way as to run enough of the code where the agent/analyzer can discover all classes accessed via reflection, etc. I then manually copy the output into the `src/main/graal` directory.

If they are not included there will be runtime (e.g. 'class not found') errors when classes are accessed via reflection. The previous PR didn't have them, but if you actually tried to run the built tool it would fail.

Reply:
I eventually want to move these files into `bitcoinj-core` and have semi- or fully automated tools to generate them. Perhaps by using the `-Pagent` option while running unit tests. By putting them in `core` any app using **bitcoinj** will be able to produce a native-image compilation of **bitcoinj** when needed.